### PR TITLE
RTC381749 - use seperate jars for jakarta

### PR DESCRIPTION
**Pull request by benjamin-confino**

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Review by Joseph-Cass — APPROVED** [review]
LGTM :)

I'm not familiar with gradle code, but it looks sensible, and the buckets ran and passed in the PB

**Comment by Joseph-Cass** [comment]
I think the copyright headers may need updating?

**Comment by benjamin-confino** [comment]
#build

File: dev/fattest.simplicity/src/componenttest/custom/junit/runner/RepeatTestFilter.java [hunk 1]
```diff
@@ -147,12 +150,12 @@ public static String getRepeatActionsAsString() {
...
-     * Is the repeat action active?
+     * Is the repeat action currently active?
      *
      * @param  action The repeat action to check.
-     * @return        True if the repeat action is active.
+     * @return True if the repeat action is active.
      */
     public static boolean isRepeatActionActive(String action) {
-        return REPEAT_ACTION_STACK.contains(action);
+        return REPEAT_ACTION_STACK.peek() == null ? false : REPEAT_ACTION_STACK.peek().equals(action);
```
Comment (tevans78) [inline dev/fattest.simplicity/src/componenttest/custom/junit/runner/RepeatTestFilter.java:159]: I can't say if this is correct or not... you need to ask around more widely

Comment (benjamin-confino) [reply dev/fattest.simplicity/src/componenttest/custom/junit/runner/RepeatTestFilter.java]: Removed for now.

**Comment by benjamin-confino** [comment]
#build

**Review by tevans78 — APPROVED** [review]

**Comment by benjamin-confino** [comment]
#rub-libby-bot

**Comment by benjamin-confino** [comment]
#run-libby-bot

**Pull request merged**